onnx/backend/test/data/real/test_bvlc_alexnet/data.json

```diff
@@ -1 +1 @@
-{"atol": 1e-07, "model_name": "bvlc_alexnet", "rtol": 0.001, "url": "https://s3.amazonaws.com/download.onnx/models/opset_9/bvlc_alexnet.tar.gz"}
\ No newline at end of file
+{"atol": 1e-07, "model_name": "bvlc_alexnet", "rtol": 0.001, "url": "https://s3.amazonaws.com/download.onnx/models/opset_10/bvlc_alexnet.tar.gz"}
```

Comment:
how is this change relevant to this PR?

Reply:
It is not. Probably one of the generator scripts created it

Reply:
the test generator script can update other tests (example their opset). Please only include the test changes relevant to your PR.

Reply:
fixed